components/view-switcher/test/view-switcher.test.js

```diff
@@ -0,0 +1,50 @@
+import '../view-switcher.js';
+import '../view-switcher-item-button.js';
+import { clickElem, expect, fixture, html, oneEvent, runConstructor } from '@brightspace-ui/testing';
+
+describe('d2l-view-switcher', () => {
+
+	describe('constructor', () => {
+
+		it('should construct', () => {
+			runConstructor('d2l-view-switcher');
+		});
+
+	});
+
+	let elem, items;
+	beforeEach(async() => {
+		elem = await fixture(html`
+			<d2l-view-switcher>
+				<d2l-view-switcher-item-button key="saveAsDraft" text="Save as Draft"></d2l-view-switcher-item-button>
+				<d2l-view-switcher-item-button key="saveAndClose" text="Save and Close"></d2l-view-switcher-item-button>
+				<d2l-view-switcher-item-button key="saveAndNew" text="Save and New"></d2l-view-switcher-item-button>
+			</d2l-view-switcher>
+		`);
+		items = elem.querySelectorAll('d2l-view-switcher-item-button');
+	});
+
+	describe('selection', () => {
+		it('should have first item selected by default', () => {
+			expect(items[0].selected).to.be.true;
+			expect(items[1].selected).to.be.false;
+			expect(items[2].selected).to.be.false;
+		});
+		it('should fire select event when item is clicked', async() => {
+			setTimeout(() => clickElem(items[2]));
```

Comment:
Is the `setTimeout` needed? I _think_ this can probably just be `clickElem(items[2])`. Same for next test.